quickcomm/templates/quickcomm/post.html

```diff
@@ -16,6 +16,7 @@ <h5 class="card-title align-self-center">{{ post.author }} </h5>
       <p class="card-text">{{ post.content }}</p>
       {% endif %}
       <p class="card-text">{{ post.description}}</p>
+      <p class="text-muted"> Likes : {{post.likes.count}} </p>
```

Comment:
I'm getting an OperationalError when trying to view a post that I just made because of this line. It says 'no such table: quickcomm_post_likes'.

I've run makemigrations and migrate, is there anything else I should have to do for this to work?

Reply:
I just tried making a new post  and the only error I got was No reverse match for 'view_profile' which I assumed was a bug someone already had see before
But if I go to authors and try opening the new post, it works

Could you let me know the exact steps you did so I can recreate this bug?